packages/space-opera/src/components/model_viewer_snippet/model_viewer_snippet.ts

```diff
@@ -125,27 +116,28 @@ export class ExportPanel extends ConnectedLitElement {
   .open=${true} .sticky=${true} 
   .copyFunction=${this.snippetCopyToClipboard.bind(this)}>
   <div slot="content">
-    <snippet-viewer id="snippet-header" .renderedSnippet=${snippet}
-      .renderedStyle=${this.hotspots.length > 0 ? hotspotStyles.cssText : ``}>
+    <snippet-viewer id="snippet-header" .renderedSnippet=${snippet}>
```

Comment:
The snippet visible in the editor now does not show the CSS. Maybe we want some kind of note here, saying that everything is in the ZIP? (CSS for ar button, modelviewer, progress bar, hotspots)

Reply:
Yeah, or maybe even remove the copy snippet button to push people toward the zip.